terraform/deployments/eks/main.tf

```diff
@@ -22,51 +17,23 @@ provider "aws" {
   }
 }
 
-resource "aws_iam_role" "eks_cluster" {
-  name = "eks_cluster-tmp"
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "17.1.0"
 
-  assume_role_policy = <<POLICY
-{
-  "Version": "2012-10-17",
-  "Statement": [
+  cluster_name    = "govuk-tmp" # TODO: name
+  cluster_version = "1.21"
+  subnets         = data.terraform_remote_state.infra_networking.outputs.private_subnet_ids
+  vpc_id          = data.terraform_remote_state.infra_networking.outputs.vpc_id
+  manage_aws_auth = false
+
+  worker_groups = [
     {
-      "Effect": "Allow",
-      "Principal": {
-        "Service": "eks.amazonaws.com"
-      },
-      "Action": "sts:AssumeRole"
+      instance_type        = "m5.xlarge"
```

Comment:
We could parameterize this one but might be worth doing later.

Reply:
Ooh, good thing you mentioned that. Turns out we already have a var for it but I didn't substitute it here. Fixed.